goodtablesio/settings.py

```diff
@@ -24,6 +24,9 @@
 GLOB_EXCLUDED_FORMATS = ['json', 'sql']
 GOOGLE_ANALYTICS_CODE = os.environ.get('GOOGLE_ANALYTICS_CODE')
 MAX_TABLES_PER_SOURCE = 100
+DEMO_API_URL = '%s/api' % BASE_URL
+DEMO_API_TOKEN = 'D0123458B8E36326C60253FE4A7FF6662CAB0C48'
```

Comment:
We should not hardcode this in the source code, so it's easy to change. Rather read it from an env var in `settings`